rust/src/lib.rs

```diff
@@ -48,7 +77,7 @@ pub use self::initramfs::*;
 mod lockfile;
 pub use self::lockfile::*;
 mod livefs;
-pub use self::livefs::*;
+pub(crate) use self::livefs::*;
```

Comment:
Note that we only need `pub(crate)` now because it's cxx that's taking care of generating the `#[no_mangle]` C entrypoints.